packages/devtools_testing/pubspec.yaml

```diff
@@ -2,16 +2,16 @@ name: devtools_testing
 description: Package of shared testing code for Dart DevTools.
 # Note: this version should only be updated by running tools/update_version.sh
 # that updates all versions of packages from packages/devtools.
-version: 0.2.3
+version: 0.2.4
```

Comment:
don't update this version. we'll update all the versions in lock step when we publish.